README.md

```diff
@@ -1,4 +1,54 @@
-# CCC Attestation Governance Documents
+# CCC Attestation Projects
 
-A project of the [Confidential Computing
+This repo contains governance and other documents related to attestation
+projects within the [Confidential Computing
 Consortium](https://github.com/confidential-computing).
+
+
+## Observation on the need for common standards
+
+Attestation features found in current offerings in the Confidential Computing
+space meaningfully improve security of confidential applications by offering
+computing parties a way to verify specific properties of each other’s
+workloads, lessening the need for blind trust that must be placed on the other
+party. However, these solutions do not yet offer a standard way for customers
+to declare, configure and authenticate attestation claims of a peer workload
+that might be running in a peer TEE (e.g., Intel SGX, AMD SEV, ARM TrustZone,
+etc. or other protected environment), they may offer partial or limited
```

Comment:
* _"partial or limited"_ one of the two looks redundant
* adding an _"and"_ after the comma

```suggestion
etc. or other protected environment), and they may offer limited
```